src/core/highlight.js

```diff
@@ -58,7 +59,6 @@ export async function run(conf) {
         if (id !== msg.id) {
           return; // not for us!
         }
-        element.innerHTML = value;
```

Comment:
Not sure how this works, as the `value` is supposed to be the highlighted HTML? What am I missing?